documentation/asset-management/topics/asset-properties.rst

```diff
@@ -0,0 +1,93 @@
+.. include:: /asset-management/media.rst
+.. _Important-asset-properties-doc:
+
+****************
+Asset properties
+****************
+Assets can have many properties and many property types. These properties can be updated using one of the following methods:
+
+1. Discovery
+2. Inherited from catalog data
+3. Inherited from the parent asset
```

Comment:
Inherited from the parent asset at creation

Reply:
Will address. Thanks!